typo3Model.xml

```diff
@@ -41,6 +41,27 @@
       <type name="dkd:typo3:pages">
           <title>TYPO3 Page</title>
           <parent>cm:folder</parent>
+          <properties>
+              <property name="dkd:typo3:pages:doktype">
+                  <type>d:int</type>
+                  <mandatory>true</mandatory>
+                  <multiple>false</multiple>
+              </property>
+          </properties>
+          <associations>
+              <association name="dkd:typo3:pages:shortcut_to">
```

Comment:
In cmis there is and so there must be also a way in alfresco.
Check out the CMIS doc http://docs.oasis-open.org/cmis/CMIS/v1.1/csd01/CMIS-v1.1-csd01.html#x1-560006
It also allows you to define the accepted source and target type